src/Build/Globbing/Extensions/IMSBuildGlobExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//-----------------------------------------------------------------------
+
+
+using System.Collections.Generic;
+
+namespace Microsoft.Build.Globbing.Extensions
+{
+    /// <summary>
+    /// Extensions for <see cref="IMsBuildGlob"/>
+    /// </summary>
+    public static class IMsBuildGlobExtensions
```

Comment:
In my mind the class name meant to say that it contains extension methods for the `IMSBuildGlob` type. I guess that when you take a step back and look at it from a distance it does not look like a good name anymore :)